utils/stellar.js

```diff
@@ -60,6 +61,68 @@ exports.stellar = (function() {
     })
   }
 
+  createEscrowAccount = (secretKey, shaSignerKey, callback) => {
+    StellarSdk.Network.useTestNetwork();
+    let sourceKeypair = StellarSdk.Keypair.fromSecret(secretKey);
+    let escrowKeypair = StellarSdk.Keypair.random();
+    
+    (async () => {
```

Comment:
Creating a named function expression and then invoking it might make the code a bit more expressive but only adds one extra line. Something like 

```
buildAccountFromOptions { 
  try { ... }
  ...
}
buildAccountFromOptions()
```

But maybe there's a reason for not doing this. What are your thoughts on this?

Reply:
Seems like since most of the method is the use of the IIFE, that this might not add much value. Also, using the IIFE like that is a pattern I've seen in others' code when doing certain things with the Stellar-SDK